src/eventDispatcher.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+class EventDispatcher {
+    constructor() {
+        this.eventDispatcherMap = {};
+    }
+
+    bind(eventName, handler) {
+        if (typeof handler === 'function') {
+            if (this.eventDispatcherMap[eventName]) {
+                this.eventDispatcherMap[eventName].push(handler);
+            } else {
+                this.eventDispatcherMap[eventName] = [handler];
+            }
+        }
+    }
+
+    trigger(eventName) {
+        if (this.eventDispatcherMap[eventName]) {
+            for (var i = 0; i < this.eventDispatcherMap[eventName].length; i++) {
+                try {
+                    this.eventDispatcherMap[eventName][i]();
+                } catch (e) {
+
+                }
+            }
+        }
+    }
+
+    unbind(eventName, handler) {
+        if (this.eventDispatcherMap[eventName]) {
+            for (var i = 0; i < this.eventDispatcherMap[eventName].length; i++) {
```

Comment:
Instead of `var`, for local variables you re now going to use `let`. Investigate the difference please.